spec/controllers/works_controller_spec.rb

```diff
@@ -3,6 +3,25 @@
 describe WorksController do
   include LoginMacros
 
+  describe "feed" do
+    before do
+      @user = FactoryGirl.create(:user)
+      @fandom = FactoryGirl.create(:fandom, name: "Stargate")
+      @work = FactoryGirl.create(:work, posted: true, 
+                                 fandom_string: @fandom.name, 
+                                 title: "Stargate and the Methods of Security")
+      @pseud = FactoryGirl.create(:pseud, user_id: @user.id) 
+      @chapter = FactoryGirl.create(:chapter, work_id: @work.id, 
+                                    title: "Stargate and the Methods of Security", 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.